src/main.rs

```diff
@@ -122,6 +136,9 @@ struct ChecksumSource {
     /// Specify additional checksums or checksum patterns to search for
     #[arg(short = 'p', long = "pattern", value_name = "TEMPLATE")]
     checksum_patterns: Vec<String>,
+    /// Specify the checksum value for the downloaded file
```

Comment:
I assume the group macro on top prevents the use `-p` at the same time as `-H`

Reply:
Correct, it's the `multiple = false ` attibute that enforces this